### ISSUE TITLE
Unable to download the complete datasets

### ISSUE THREAD

**Issue by minjunli** [visipedia/imat_comp#2]
I downloaded the dataset at June 14th and found that around 11% of the provided images URLs already not working. The training set has 4683 missing images, validation set has 919 and testing set has 3728 missing images. Filename of the missing images are listed below:
[valid_missing.txt](https://github.com/visipedia/imat_comp/files/1080955/valid_missing.txt)
[train_missing.txt](https://github.com/visipedia/imat_comp/files/1080956/train_missing.txt)
[test_missing.txt](https://github.com/visipedia/imat_comp/files/1080957/test_missing.txt)

During the first few days, about 2% images of the testing set can not be accessed according to  [this post](https://www.kaggle.com/c/imaterialist-challenge-FGVC2017/discussion/34120). And currently around 11% of the testing images URLs are not accessible.

Is there any ways to solve this problem?

**Comment by richardaecn** [comment]
Hi, we are currently working on this issue and thinking of putting images online. We will let you know if there is any update.

**Comment by richardaecn** [comment]
Hi, I've added a download link in the github page.

**Comment by minjunli** [comment]
Thanks a lot!
